docs/gitlab.md

```diff
@@ -41,6 +41,6 @@ git config --global credential.gitlabauthmodes browser
 
 Improved support requires changes in GitLab. Please vote for these issues if they affect you:
 
-1. No support for OAuth device authorization (necessary for machines without web browser) https://gitlab.com/gitlab-org/gitlab/-/issues/332682
-2. Only domains with prefix `gitlab.` are recognised as GitLab remotes https://gitlab.com/gitlab-org/gitlab/-/issues/349464
-3. Username/password authentication is suggested even if disabled on server https://gitlab.com/gitlab-org/gitlab/-/issues/349463
+1. No support for OAuth device authorization (necessary for machines without web browser): [GitLab issue 332682](https://gitlab.com/gitlab-org/gitlab/-/issues/332682)
```

Comment:
Should we have short names for the links in this document similar to those added in `CODE_OF_CONDUCT.md`?